migrations/versions/12780a039923_populate_database.py

```diff
@@ -0,0 +1,82 @@
+"""populate database
+
+Revision ID: 12780a039923
+Revises: 67ac3b7d5c2f
+Create Date: 2021-09-14 14:44:50.802230
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '12780a039923'
+down_revision = '67ac3b7d5c2f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('profile',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=500), nullable=True),
+    sa.Column('first_name', sa.String(), nullable=True),
+    sa.Column('last_name', sa.String(), nullable=True),
+    sa.Column('avatar_url', sa.String(length=500), nullable=True),
+    sa.Column('bio_rendered', sa.String(), nullable=True),
+    sa.Column('interests', sa.String(), nullable=True),
+    sa.Column('before_rc', sa.String(), nullable=True),
+    sa.Column('during_rc', sa.String(), nullable=True),
```

Comment:
Definitely some of these should be nullable, but I'd be curious if we can tighten any of these up. In the real data, do we always have `avatar_url`? Or `name`?

Reply:
Your're right, `name` should not be nullable. This ended up being the only consistent attribute between `user` and `profile` (the old `user` table doesn't have first and last names) so that is the only nullable attribute I found.

Reply:
Awesome, thanks for checking!